keps/sig-cloud-provider/providers/0021-cloud-provider-digitalocean.md

```diff
@@ -54,23 +54,23 @@ The existing repository hosting the [DigitalOcean cloud controller manager](http
 
 ### User Experience Reports
 
-DigitalOcean recently announced a [Kubernetes offering](https://www.digitalocean.com/products/kubernetes/). Many users have already signed up for early access. DigitalOcean is also a gold member of the CNCF.
+DigitalOcean recently announced a [Kubernetes offering](https://www.digitalocean.com/products/kubernetes/). Many users have already signed up for early access.
 
 ### Testgrid Integration
 
-TODO
+DigitalOcean uploads conformance test results for every patch release to testgrid. You can see the results [here](https://k8s-testgrid.appspot.com/conformance-cloud-provider-digitalocean#Summary).
 
 ### CNCF Certified Kubernetes
 
-TODO
+DigitalOcean is a gold member of the CNCF.
```

Comment:
I suggest omitting level, to avoid perception that higher levels of membership implies higher quality or different features.

Reply:
Being someone who runs a trademark program, there's a distinction between being "Certified Kubernetes" and passing conformance. The latter is a requirement for the former, but unless you're a non-profit you have to purchase a membership as part of the certification process (this is similar to how the program I administer is run). In the spirit of openness and a level playing field, it might make sense to distinguish between the two completely.

Reply:
Although we very much appreciate DigitalOcean's gold membership in CNCF, I think it would be more appropriate in this section to say that "DigitalOcean's Certified Kubernetes offering is underway." You could also add that several other Certified Kubernetes offerings are deployed on top DigitalOcean: https://github.com/cncf/k8s-conformance/search?q=digitalocean&unscoped_q=digitalocean